release/nightly_tests/dataset/multi_node_read_images_benchmark_compute.yaml

```diff
@@ -0,0 +1,15 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
+
+max_workers: 19
+
+head_node_type:
+    name: head_node
+    instance_type: m5.16xlarge
+
+worker_node_types:
+    - name: worker_node
+      instance_type: m5.4xlarge
+      max_workers: 19
+      min_workers: 19
```

Comment:
this test only fetches metadata. I think we don't need this many nodes? I guess 8 nodes should be enough.

Reply:
The test also iterates over the data:

```python
ds = ray.data.read_images(hundred_million_image_paths)
for _ in ds.iter_batches(batch_size=None, batch_format="pyarrow"):
    pass
```

20 nodes was somewhat arbitrary. It's the amount we use for the data ingest benchmark. Let's tune the value once we get the test running in CI.